docs/interactions/Receiving_and_Responding.md

```diff
@@ -90,6 +90,9 @@ This is sent on the [message object](#DOCS_RESOURCES_CHANNEL/message-object) whe
 | type | [interaction type](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-interaction-type) | the type of interaction                                                                                                                 |
 | name | string                                                                                              | the name of the [application command](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-structure) |
 | user | [user object](#DOCS_RESOURCES_USER/user-object)                                                     | the user who invoked the interaction                                                                                                    |
+| member?* | [partial member](#DOCS_RESOURCES_GUILD/guild-member-object) objects                             | the member who invoked the interaction in the guild                                                                                       |
+
+\* Partial `Member` objects only have `roles`, `mute`, `joined_at`, `hoisted_role` and `deaf` fields. Only sent in `MESSAGE_CREATE` [Gateway Event](#DOCS_TOPICS_GATEWAY/message-create)
```

Comment:
Are you sure these are all the fields? While testing this, I got a few more fields.
```json
{
   "interaction":{
      "id":"933457462147813387",
      "type":2,
      "name":"test",
      "user":{
         "username":"Victorsitou",
         "public_flags":64,
         "id":"184040656761651200",
         "discriminator":"7165",
         "avatar":"a_84302c11d747e16cac1c646bb8efcf32"
      },
      "member":{
         "roles":[
            "927583695240593470"
         ],
         "premium_since":null,
         "pending":false,
         "nick":null,
         "mute":false,
         "joined_at":"2021-05-20T02: 46: 51.470000+00: 00",
         "hoisted_role":"927583695240593470",
         "deaf":false,
         "communication_disabled_until":null,
         "avatar":null
      }
   }
}
```

Reply:
Added